### ISSUE TITLE
maxpool后会高度为0

### ISSUE THREAD

**Issue by afterimagex** [WenmuZhou/PytorchOCR#119]
https://github.com/WenmuZhou/PytorchOCR/blob/4a4f705c663dbc60563f1921b66415a953fbe12d/torchocr/networks/backbones/RecMobileNetV3.py#L136

```
if __name__ == '__main__':
    import torch
    from torchsummary import summary

    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
    net = MobileNetV3(3, scale=0.5, model_name='small').to(device)
    # print(net)
    summary(net, input_size=(3, 32, 320))
```

会报错，我改成`self.pool = nn.MaxPool2d(kernel_size=(1, 2), stride=(1, 2), padding=0)`可以运行

**Comment by novioleo** [comment]
@afterimagex 那你这不就是没有对x轴的值进行pool了吗

**Comment by afterimagex** [comment]
> @afterimagex 那你这不就是没有对x轴的值进行pool了吗

我也很奇怪呀，难道是我的使用方式不对吗？

**Comment by novioleo** [comment]
为0的情况理论上就是高度不够了~所以你算一下下采样的倍率 @afterimagex
